scripts/ci/osx/build.sh

```diff
@@ -3,3 +3,19 @@ set -ev
 echo "Building openFrameworks - OSX Template Project"
 ROOT=${TRAVIS_BUILD_DIR:-"$( cd "$(dirname "$0")/../../.." ; pwd -P )"}
 xcodebuild -configuration Release -target emptyExample -project "$ROOT/scripts/osx/template/emptyExample.xcodeproj"
+
+echo "Unit tests"
+cd $ROOT/tests
+for group in *; do
+	if [ -d $group ]; then
+		for test in $group/*; do
+			if [ -d $test ]; then
+				cd $test
+				cp ../../../scripts/osx/template/Makefile .
+				cp ../../../scripts/osx/template/config.make . 
+				make Debug
```

Comment:
@arturoc wouldn't it be better to also use xcodebuild for this (or make for the main compilation) to have homogenity in the build system per platform?